sled-agent/src/rack_setup/service.rs

```diff
@@ -1239,14 +1272,17 @@ impl ServiceInner {
                 matches!(zone_type, OmicronZoneType::InternalDns { .. })
             },
         );
+        rss_step.update(RssStep::ConfigureDNS);
```

Comment:
I think `InitDns` and `ConfigureDns` are backwards - at this step we're waiting for the DNS zones to exist, and the next step is sending them their configs.

Reply:
done